src/Microsoft.AspNetCore.Blazor.Browser.JS/.gitignore

```diff
@@ -1,2 +1,4 @@
 ﻿node_modules/
 dist/
+src/**/*.js
+src/**/*.js.map
```

Comment:
Out of interest, what causes you to get `.js`/`.js.map` files in this directory? VS shouldn't be dropping any such files there because we have `<TypeScriptCompileBlocked>true</TypeScriptCompileBlocked>` in the csproj.

So hopefully these changes aren't required.

Reply:
It's good to have it as the MSBuild code to disable quick check is not need it, and if you just run tsc from the command line, this madness happens and it's hard to clean up.